ouroboros-consensus/docs/Contributing.md

```diff
@@ -1,5 +1,35 @@
-TODO eg git history goals etc
+# How to contribute to the Consensus Layer
 
-See the
-https://github.com/input-output-hk/ouroboros-network/ouroboros-consensus/docs/StyleGuide.md
-document.
+See the follow documents.
+
+  * this repository's [README][repo-README]; it links to design documents
+  * Consensus [style guide][consensus-style]
+  * Consensus [git process][consensus-git]
+  * Consensus [onboarding][consensus-onboarding]
+  * TODO quick-start guide
+
+Our primary neighbors are the upstream [cardano-ledger-specs] and the downstream
+[cardano-node].
+
+If you're not an IOHK employee, please fork the repository on GitHub and open
+PRs that way. We're excited about your involvement; thank you!
```

Comment:
I think this paragraph should be moved to the top of the file. Encourage the contribution, ground general expectations to the soon-to-be contributor.

Reply:
Great!